Azure/sonic-buildimage
# Fix issue test_crm and test_fib

#### Why I did it
Fix issue (https://github.com/Azure/sonic-buildimage/issues/9171) and (https://github.com/Azure/sonic-buildimage/issues/9236)

#### How I did it
Add flag to config file to get correct IPv6 entry count.
Add initialization configuration file to set IPv4 ECMP hash on L4.
#### How to verify it
Compile the sonic_platform wheel for e1031, then upload to device and install the wheel, verify using testbed.
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
